src/wiki/forms.py

```diff
@@ -285,12 +280,15 @@ def clean(self):
         return cd
 
 
-class SelectWidgetBootstrap(BuildAttrsCompat, forms.Select):
+class SelectWidgetBootstrap(forms.Select):
     """
     http://twitter.github.com/bootstrap/components.html#buttonDropdowns
     Needs bootstrap and jquery
     """
 
+    template_name = "wiki/forms/select.html"
+    option_template_name = "wiki/forms/select_option.html"
```

Comment:
I can imagine this must have been confusing with the currently un-designed structure in the `wiki/templates` folder. On one hand, they could live in `wiki/templates/wiki/includes`, and on the other hand, this folder already has form components for the bootstrap template tag sitting in its root, so that would *also* be confusing.

I guess all of this is pending a refactor at some point... but should preferably be announced a bit in advance, since a customized theme can be a real pain to update if we change template structures.

Reply:
Well, we can easily refactor until 0.4 "stable" release :joy_cat: